infra/resources/prod/function_lollipop.tf

```diff
@@ -0,0 +1,190 @@
+}
+
+locals {
+  function_lollipop = {
+    name = "lollipop"
+
+    app_settings = {
+      NODE_ENV = "production"
+
+      // Keepalive fields are all optionals
+      FETCH_KEEPALIVE_ENABLED             = "true"
+      FETCH_KEEPALIVE_SOCKET_ACTIVE_TTL   = "110000"
+      FETCH_KEEPALIVE_MAX_SOCKETS         = "40"
+      FETCH_KEEPALIVE_MAX_FREE_SOCKETS    = "10"
+      FETCH_KEEPALIVE_FREE_SOCKET_TIMEOUT = "30000"
+      FETCH_KEEPALIVE_TIMEOUT             = "60000"
+
+      COSMOSDB_NAME                = "citizen-auth"
+      COSMOSDB_URI                 = data.azurerm_cosmosdb_account.cosmos_citizen_auth.endpoint
+      COSMOSDB_KEY                 = data.azurerm_cosmosdb_account.cosmos_citizen_auth.primary_key
+      COSMOS_API_CONNECTION_STRING = format("AccountEndpoint=%s;AccountKey=%s;", data.azurerm_cosmosdb_account.cosmos_citizen_auth.endpoint, data.azurerm_cosmosdb_account.cosmos_citizen_auth.primary_key)
+
+      #TODO: move to new storage on itn
+      LOLLIPOP_ASSERTION_STORAGE_CONNECTION_STRING = data.azurerm_storage_account.lollipop_assertion_storage.primary_connection_string
+      LOLLIPOP_ASSERTION_REVOKE_QUEUE              = "pubkeys-revoke-v2"
+
+      // ------------
+      // JWT Config
+      // ------------
+      ISSUER = local.lollipop_jwt_host
+
+      PRIMARY_PRIVATE_KEY = trimspace(data.azurerm_key_vault_certificate_data.lollipop_certificate_v1.key)
+      PRIMARY_PUBLIC_KEY  = trimspace(data.azurerm_key_vault_certificate_data.lollipop_certificate_v1.pem)
+
+      // Use it during rotation period. See https://pagopa.atlassian.net/wiki/spaces/IC/pages/645136398/LolliPoP+Procedura+di+rotazione+dei+certificati
+      //SECONDARY_PUBLIC_KEY =
+
+      // -------------------------
+      // First LolliPoP Consumer
+      // -------------------------
+      FIRST_LC_ASSERTION_CLIENT_BASE_URL         = "https://api.io.pagopa.it"
+      FIRST_LC_ASSERTION_CLIENT_SUBSCRIPTION_KEY = data.azurerm_key_vault_secret.first_lollipop_consumer_subscription_key.value
+    }
+  }
+}
+
+resource "azurerm_resource_group" "function_lollipop_rg" {
+  name     = "${local.project}-${local.domain}-${local.function_lollipop.name}-rg-02"
+  location = local.location
+
+  tags = local.tags
+}
+
+module "function_lollipop" {
+  source = "github.com/pagopa/dx//infra/modules/azure_function_app?ref=ab26f57ed34a614fd3fa496c7b521be9ecc88e1b"
+
+  environment = {
+    prefix          = local.prefix
+    env_short       = local.env_short
+    location        = local.location
+    domain          = local.domain
+    app_name        = local.function_lollipop.name
+    instance_number = "02"
+  }
+
+  resource_group_name = azurerm_resource_group.function_lollipop_rg.name
+  health_check_path   = "/info"
+  node_version        = 20
+  # P2mv3 SKU and 8 Worker process count
+  tier = "xl"
+
+  subnet_cidr   = local.cidr_subnet_fn_lollipop
+  subnet_pep_id = data.azurerm_subnet.private_endpoints_subnet.id
+
+  subnet_service_endpoints = {
```

Comment:
evaluate to remove this settings

Reply:
removed in https://github.com/pagopa/io-auth-n-identity-domain/pull/132/commits/e4d5135b95b33dec8ea56597d13eb3f56032dca0